### PR TITLE
Feature/bitrise circle ci hybrid

### LINKED ISSUE #2707: Enable Bitrise CI and CircleCI Release
Currently there is an issue with how Bitrise generates the apps for release (not accepting biometric on app upgrade requiring the user to login with their password).

Enable Bitrise for CI and Nightly E2E
Use CircleCI for Deployment to TestFlight

### PR DESCRIPTION
**Pull request by sethkfman**
**Description**

Create a hybrid build system that uses Bitrise for CI & CircleCI for deployments

**Checklist**

* [X] There is a related GitHub issue
* [X] Tests are included if applicable
* [X] Any added code is fully documented

**Issue**

Resolves #2707

### CODE REVIEW

**Comment by sethkfman** [comment]
I will move the required build check from CircleCI to the Bitrise once the code review has dev sign-off.

File: scripts/build.sh [hunk 1]
```diff
@@ -218,11 +223,11 @@ buildAndroidRelease(){
 	prebuild_android
 
 	if [ "$PRE_RELEASE" = true ] ; then
-		TARGET="android/app/build.gradle"
-		sed -i'' -e 's/getPassword("mm","mm-upload-key")/"ANDROID_KEY"/' $TARGET;
-		sed -i'' -e "s/ANDROID_KEY/$ANDROID_KEY/" $TARGET;
-		echo "$ANDROID_KEYSTORE" | base64 --decode > android/keystores/release.keystore
-	fi
+ 		TARGET="android/app/build.gradle"
```
Comment (rickycodes) [inline scripts/build.sh:226]: I don't think these changes are needed?

**Review by rickycodes — APPROVED** [review]
a couple of small suggestions, but nothing to block over

File: .circleci/config.yml [hunk 2]
```diff
@@ -195,36 +195,37 @@ jobs:
...
-              ignore:
-                - main
-                - develop
-      - lint:
-          requires:
-            - prep-node-deps
-      - test-unit:
-          requires:
-            - prep-node-deps
+      # - prep-node-deps:
```
[the remark this answers is not in the document]
Comment (sethkfman) [reply .circleci/config.yml:208]: I like that thought process

File: scripts/build.sh (hunk 1 above)
Comment (sethkfman) [reply scripts/build.sh]: reverted

**Pull request merged**